src/modules/ticket/reducer.ts

```diff
@@ -0,0 +1,89 @@
+import {Alert} from 'react-native';
+import {Action} from 'redux';
+import {t} from '@translations';
+
+export const BUY_TICKET_REQUEST = 'BUY_TICKET_REQUEST';
+export const BUY_TICKET_SUCCESS = 'BUY_TICKET_SUCCESS';
+export const BUY_TICKET_FAIL = 'BUY_TICKET_FAIL';
+
```

Comment:
Constant variables on `Line:5` to `Line:7` should be moved to a separate file inside `modules/ticket`. It an be called something like `ActionConstant.ts`